frontend/src/app/components/address/address-preview.component.ts

```diff
@@ -64,13 +64,15 @@ export class AddressPreviewComponent implements OnInit, OnDestroy {
           this.address = null;
           this.addressInfo = null;
           this.addressString = params.get('id') || '';
-          if (/^[A-Z]{2,5}1[AC-HJ-NP-Z02-9]{8,100}$/.test(this.addressString)) {
+          if (/^[A-Z]{2,5}1[AC-HJ-NP-Z02-9]{8,100}|[A-F0-9]{130}$/.test(this.addressString)) {
```

Comment:
The length is 130 or 66. There are some people who have spent to compressed p2pk.

To be specific it would be `(04[A-F0-9]{128}|(02|03)[A-F0-9]{64})`

We could just ignore compressed p2pk though.